src/library/scala/collection/Seq.scala

```diff
@@ -845,6 +865,26 @@ trait SeqOps[+A, +CC[_], +C] extends Any
 }
 
 object SeqOps {
+  /** Operations for comparing the length of a collection to a test value.
+    *
+    * These operations are implemented in terms of
+    * [[scala.collection.SeqOps.lengthCompare(Int) `lengthCompare(Int)`]].
+    */
+  final class LengthCompareOps[+A, +CC[_], +C] private[SeqOps](val seq: SeqOps[A, CC, C]) extends AnyVal {
```

Comment:
We don’t really need these type parameters, we could just take a `SeqOps[_, _, _]` as parameter.

Reply:
unfortunately, `CC` takes a type parameter, and `_[_]` is not allowed (yet?).

I can reduce it to `LengthCompareOps[+CC[_]](val seq: SeqOps[_, CC, _])`, however. Does that still  sound good?

Reply:
@julienrf

Reply:
You can use `scala.collection.AnyConstr` in place of the `CC[_]` type parameter.

Reply:
I keep forgetting about that. Thanks!